deps.ts

```diff
@@ -12,10 +12,4 @@ export {
   setCookie,
 } from "https://deno.land/std@0.201.0/http/cookie.ts";
 export { assert } from "https://deno.land/std@0.201.0/assert/assert.ts";
-export {
-  OAuth2Client,
-  type OAuth2ClientConfig,
-  OAuth2ResponseError,
-  type Tokens,
-} from "https://deno.land/x/oauth2_client@v1.0.2/mod.ts";
```

Comment:
Removed because nothing else requires this other than the _internal/oauth2_client.ts module